build_docs.py

```diff
@@ -200,9 +200,10 @@ def build_one(version, isdev, quick, venv, build_root, www_root,
     logname = "{}-{}.log".format(os.path.basename(checkout), language)
     python = os.path.join(venv, "bin/python")
     sphinxbuild = os.path.join(venv, "bin/sphinx-build")
+    blurb = os.path.join(venv, "bin/blurb")
     shell_out(
-        "cd Doc; make PYTHON=%s SPHINXBUILD=%s SPHINXOPTS='%s' %s >> %s 2>&1" %
-        (python, sphinxbuild, sphinxopts, maketarget,
+        "cd Doc; make PYTHON=%s SPHINXBUILD=%s BLURB=%s VENVDIR=%s SPHINXOPTS='%s' %s >> %s 2>&1" %
```

Comment:
Why do you also need `VENVDIR`?

Reply:
"Explicit is better than implicit"  In case it ever gets changed.

Reply:
But if the venv dir gets changed all the binary paths will break.